db/schema.rb

```diff
@@ -29,7 +29,7 @@
     t.datetime "updated_at"
   end
 
-  create_table "docker_registries", force: :cascade do |t|
+  create_table "registries", force: :cascade do |t|
```

Comment:
Shouldn't this be done via a new migration? This will break people upgrading from 2.0, no?